### PR TITLE
Setup Axios GET call

### PR DESCRIPTION
**Pull request by tlj4k8**
## Setup Axios GET call

- Installed axios `npm i axios`
- Created a working example

Used this fun free api - https://dog.ceo/dog-api/documentation/random
Colors inspiration - https://www.codeofcolors.com/halloween-colors.html 👻

### CODE REVIEW

**Review by LaDfBC — APPROVED** [review]
Looks fine to me.

File: hacktober-web/src/components/Home.vue [hunk 1]
```diff
@@ -20,22 +25,20 @@ export default {
 
 <!-- Add "scoped" attribute to limit CSS to this component only -->
 <style scoped>
-.home {
+.hack-home {
   display: flex;
   flex-direction: column;
   justify-content: center;
   height: 100vh;
-  width: 100vw;
```
Comment (iceicelogan) [inline hacktober-web/src/components/Home.vue:28]: For my own knowledge, why are you removing width Params?

**Review by iceicelogan — APPROVED** [review]
I like it! I don't get the width removal, but everything else is Gucci. Can't wait to click for many dogs 🐶🐕🐩🦮🐕‍🦺

File: hacktober-web/src/components/Home.vue (hunk 1 above)
Comment (tlj4k8) [reply hacktober-web/src/components/Home.vue:28]: @iceicelogan  It was causing a weird layout shift and everything was still fine without it. Really it wasn't needed.

**Pull request merged**